### ISSUE TITLE
Apple TV uses Xbox button?

### ISSUE THREAD

**Issue by twilightgod** [moonlight-stream/moonlight-ios#554]
**Is your feature request related to a problem? Please describe.**
In Apple TV, Xbox button in Xbox controller will open side control in tvOS, even in Moonlight.
I remap Xbox button in rewasd, so I can’t use this function 

**Describe the solution you'd like**
Somehow disable system level tvOS Xbox button, so that it can be used in Moonlight

**Describe alternatives you've considered**

**Additional context**

**Comment by roundaboutluke** [comment]
You can't hook it on tvOS so it's not something the devs can fix.

**Comment by parrydigm** [comment]
Like roundaboutluke said, this isn’t a Moonlight problem but rather an Apple-thinks-it-knows-best problem. They’ve nabbed certain controls for TVOS (which I can kind of understand). You may like to submit a [feature request](https://www.apple.com/feedback/) to Apple suggesting something like press-hold or triple-tap passes the Xbox Guide button to the host OS, but I wouldn’t rely on that strategy for long-term life fulfilment. 🤣

**Comment by Quack4321** [comment]
Is there still no way to fix this?

**Comment by joeblack2k** [comment]
Still no fix? Start+select at the same time ? Click both joysticks?

**Comment by oleost** [comment]
Yeah, should be a easy way in the menu to button remap. 

Example under Input option:
Remap Guide button to start+select
Or
Remap Guide button to Share button(Xbox)

**Comment by HunterAhlquist** [comment]
> Still no fix? Start+select at the same time ? Click both joysticks? 

Yeah, I don't see why there cant be a shortcut to send a home button input. PS remote play has something similar.

**Comment by anacierdem** [comment]
I think currently start+select sends alt+tab, I wish it could be configurable somehow to also support home key
